### PR TITLE
Imports syntax

### PR DESCRIPTION
**Pull request by cfnelson**
#### NOTE: Please review and test with example apps. If happy please give 👍  so that we can merge this PR. 😄

This PR was worked/tested by Aryan & Charles. We would like others to review code and test with the examples. 
- This PR moves to es6 syntax and es6 imports for meteor 1.3+.
- This PR is meant to be backwards compatible. 
- FlowRouter Example now demonstrates es6 style imports. 
- IronRouter example has been left with the old globally defined variables.

**_NOTE:**_ This PR does not resolve the needed code style improvements. This will come in a patch update.
#### TODO:
- [x] meteor-analytics.js needs to be split up
- [x] Bump the packages minor version.
- [x] Add comments to explain the imports syntax and other changes.
- [x] remove npm depends for underscore and refactor.
#### NOTE:

Any ad-blocking web-browser extension may block the entire "okgrow:analytics" package due to the word "analytics" being used in the package name. This only occurs when running Meteor in development mode as the files are not bundled. To work around this issue run your application in production mode (see below).

`meteor run --production --settings settings.json`

import syntax to import the package: 

`import { analytics } from "meteor/okgrow:analytics";`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ Use one API thanks to Segment.io's [analytics.js](https://segment.com/docs/libra
 * Quantcast
 * Segment.io
 
+## Development & Testing
+
+NOTE: Any ad-blocking web-browser extension may block the entire "okgrow:analytics" package due to the word "analytics" being used in the package name. This only occurs when running Meteor in development mode as the files are not bundled. To work around this issue run your application in production mode (see below).
```
Comment (rgould) [inline README.md]: I would maybe instead suggest that they disable their ad blocker for development (usually they ad blockers have white-list rules to allow this)

**Comment by cfnelson** [comment]
@rgould Thanks, this should be worded better. It was meant to be a warning to developers who might test with the ad-blocker on and off. Didn't want dev's to raise an issue saying their whole app doesn't work without realising it's only  an issue when in `--development` mode and it is safe to deploy to production.

**Comment by jamielob** [comment]
Some ES6 refactoring can happen here:
https://github.com/okgrow/analytics/blob/e5bead932dea3e15689f12351aa5734831ff1355/examples/flow-router/imports/flow-router.js#L34

Because you're using arrow functions on the analytics calls, they can just use `this` since the context gets passed up to the parent function.  You can then remove `self`.

**Comment by jamielob** [comment]
@cfnelson I've done some tidying up and also updated the Meteor release.  Take a quick look tomorrow and then I'll merge.

**Comment by arrygoo** [comment]
I think @jamielob's changes are good to merge as long as the examples work!

**Comment by cfnelson** [comment]
@jamielob added a few things, tested on Iron Router & Flow Router. Working nicely for me. Will merge soon after the SETTINGS.

**Pull request merged**